OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -187,7 +196,7 @@ public Aircraft(ActorInitializer init, AircraftInfo info)
 
 			// TODO: HACK: This is a hack until we can properly distinguish between airplane and helicopter!
 			// Or until the activities get unified enough so that it doesn't matter.
-			IsPlane = !info.CanHover;
+			IsPlane = !info.VTOL;
```

Comment:
If you prefer to get rid of it now, I could replace `IsPlane` with `!info.VTOL` everywhere, yes. 
It just felt like bit of a wasted effort, since that would mean changing a lot of lines that will be removed entirely once we merge `Fly` with `HeliFly` etc., because most of these look-ups will shift from `Aircraft` to the respective activities then.